config.yaml

```diff
@@ -0,0 +1,43 @@
+facility: sandbox
+log_level: debug
+namespace: default
+store_kind: fleetdb
+endpoints:
+  conditionorc:
+    authenticate: true
```

Comment:
probably should default to `false` for the sandbox case. Same comment for the `fleetdb` stanza.

Reply:
default to true, so you have to explicitly set to false and not use Auth. do not want a fleetdb incident.

Reply:
oh wait this is just the dummy config. Not the one in the helm chart. Nvm. I will change that.